inference-engine/include/cpp/ie_cnn_network.h

```diff
@@ -199,6 +199,22 @@ class INFERENCE_ENGINE_API_CLASS(CNNNetwork) {
      */
     void serialize(const std::string& xmlPath, const std::string& binPath = {}) const;
 
+    /**
+     * @brief Serialize network to IR and weights streams.
+     *
+     * @param xmlBuf output IR stream.
+     * @param binBuf output weights stream.
+     */
+    void serialize(std::ostream& xmlBuf, std::ostream& binBuf) const;
+
+    /**
+     * @brief Serialize network to IR stream and weights Blob::Ptr.
+     *
+     * @param xmlBuf output IR stream.
+     * @param binBlob output weights Blob::Ptr.
+     */
+    void serialize(std::ostream& xmlBuf, Blob::Ptr& binBlob) const;
```

Comment:
Do we need both overloads ?

Reply:
Why do we need serialization to blob?

Reply:
Because we have corresponding ReadNetwork with Blob